elements/pfe-tabs/demo/pfe-tabs.js

```diff
@@ -1,46 +1,81 @@
 import '@patternfly/pfe-band';
-import '@patternfly/pfe-button';
-import '@patternfly/pfe-jump-links';
-import '@patternfly/pfe-select';
 import '@patternfly/pfe-tabs';
+import '@patternfly/pfe-tooltip';
 
 const root = document.querySelector('[data-demo="pfe-tabs"]')?.shadowRoot ?? document;
 
-const btn = root.querySelector('#addTabAndPanelBtn');
-const pfeTabs = root.querySelector('#dynamic');
+root.querySelector('pfe-tabs');
 
-const pfeTabLabels = pfeTabs.querySelectorAll('pfe-tab');
-let pfeTabCount = pfeTabLabels.length;
+const switchToAriaDisabledInput = root.querySelector('input[name="toggle-disabled"]');
+function disabledToggle() {
+  const disabledExample = root.querySelector('#disabledExample');
+  if (switchToAriaDisabledInput.checked) {
+    disabledExample.setAttribute('active-key', '3');
```

Comment:
is `active-key` how they do it in pfv4? i'd prefer `active-index` or even better, just put `active` on the active tab pair, but expose `.activeIndex`. WDYT?

Reply:
i mean that PfeTabs.prototype.activeIndex should have a getter/setter pair, and update the element as expected, but it shouldn't reflect to or read from an attr, rather, users should just set `active` on the tab they want.

Reply:
Ok gotcha, will explore that!